SimpleTDDCsharpTest/Date_Object_Test.cs

```diff
@@ -0,0 +1,93 @@
+using Calculator;
+
+namespace SimpleTDDCsharpTest
+{
+	public class Date_Object_Test
+	{
+        private readonly Date_Object _date_Object;
+        private readonly Mock<IDNS> _dNS;
+        public Date_Object_Test()
+        {
+            _dNS =new Mock<IDNS>();
+            _date_Object =new  Date_Object(_dNS.Object);
+        }
+
+        [Fact]
+        public void Test_DNSService_Mocking_With_SendDNS()
```

Comment:
please follow best practices for naming test https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices